models/MonteCarlo.py

```diff
@@ -0,0 +1,142 @@
+from Model import Model
+from environment import Environment
+import numpy as np
+import pandas as pd
+import random
+import matplotlib.pyplot as plt
+
+'''
+A monte carlo model with epsilon greedy policy.
+
+- Episode: A complete run of the model from the start to the end of the training data.
+- Epsilon: The probability of choosing a random action instead of the best action for exploring instead of exploiting.
+- Portfolio Value: init to be 100 at first and then it will be updated based on the reward on each episode to track the value.
+
+'''
+
+class MonteCarloRL(Model):
+    def __init__(self, cleaned_data, actions, n_episodes=1000, epsilon=0.1):
+        super().__init__(state_space=None, action_space=actions)
+        self.cleaned_data = cleaned_data
+        self.actions = actions
+        self.n_episodes = n_episodes
+        self.epsilon = epsilon
+        self.nasdaq_train, self.msci_train, self.nasdaq_test, self.msci_test = self.split_data()
+    
+    @staticmethod
+    def calculate_reward(action, nasdaq_return, msci_return):
```

Comment:
could we use the environment class' get reward function? if im remembering correctly we're not using the portfolio return as a reward, but rather the log return / sharpe ratio